incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+	"time"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+
+		for _, ns := range nses {
+			runSuccess("kubectl hns set", ns, "-a")
+			//remove subname annotation
+			runSuccess("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+			runSuccess("kubectl delete ns", ns)
+		}
+
+		runSuccess("kubectl apply -f", os.Getenv("HNC_REPAIR"))
+		GinkgoT().Log(time.Now())
```

Comment:
Either take this out if it was just for short-term debugging, or add some text like `Log("Time finished:", time.Now())`. Ditto on line 34

Reply:
Removed